app/src/main/java/org/schabi/newpipe/player/PopupVideoPlayer.java

```diff
@@ -969,5 +1092,43 @@ private boolean handleMultiDrag(final MotionEvent event) {
 
             return true;
         }
+
+        /*//////////////////////////////////////////////////////////////////////////
+        // Utils
+        //////////////////////////////////////////////////////////////////////////*/
+
+        private float map(float value, float start1, float stop1, float start2, float stop2) {
```

Comment:
Might be a good idea to add some comments here to describe what map does as well as some explanations for the constants `0x88` and `1.15f` used below as well.

Reply:
The`map()` was removed, and comment made about the percentage.